### PR TITLE
Remove unused Jenkins and e2e scripts

### PR DESCRIPTION
**Pull request by jcaianirh**
Remove unused Jenkins and e2e scripts, and adjust the README.md. Also remove logging to console and add to a file called browser.log which resides in the gui_test_screenshots folder and gets copied to artifacts.

### CODE REVIEW

**Comment by jcaianirh** [comment]
@TheRealJon moved the console logging to a file as we spoke about.

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -56,23 +58,15 @@ export const config: Config = {
     }
   },
   onComplete: async() => {
-    console.log('BEGIN BROWSER LOGS');
+    const consoleLogStream = createWriteStream('gui_test_screenshots/browser.log', { flags: 'a' });
+    this.logToFile = function() {
```
Comment (TheRealJon) [inline frontend/integration-tests/protractor.conf.ts]: Could probably just move this logic inline since it's only used in one place.

**Pull request merged**